views/js/controller/creator/views/section.js

```diff
@@ -539,7 +539,10 @@ define([
                         title: defaults().sectionTitlePrefix,
                         index: 0,
                         sectionParts,
-                        visible: true
+                        visible: true,
+                        itemSessionControl: {
+                            maxAttempts: defaults().maxAttempts
```

Comment:
**chore:** `defaults()` is a function that returns a config... It was already called earlier (line 539), why not cache its results to avoid calling it each time a property is needed to be read??